visualization/app/codeCharta/ui/fileChooser/fileChooser.e2e.ts

```diff
@@ -0,0 +1,39 @@
+import { goto, newPage, launch } from "../../../puppeteer.helper"
+import { FileChooserPageObject } from "./fileChooser.po"
+import { FilePanelPageObject } from "../filePanel/filePanel.po"
+
+jest.setTimeout(60000)
+
+describe("FileChooser", () => {
+	let browser, page
+	let fileChooser: FileChooserPageObject
+	let filePanel: FilePanelPageObject
+
+	beforeAll(async () => {
+		browser = await launch()
+	})
+
+	afterAll(async () => {
+		await browser.close()
+	})
+
+	beforeEach(async () => {
+		page = await newPage(browser)
+		fileChooser = new FileChooserPageObject(page)
+		filePanel = new FilePanelPageObject(page)
+
+		await goto(page)
+	})
+
+	it("should load cc.json file", async () => {
+		await fileChooser.openFile("./app/codeCharta/assets/sample2.cc.json")
+
+		expect(await filePanel.getSelectedName()).toEqual("sample2.cc.json")
+	})
+
+	it("should not load non json file", async () => {
+		await fileChooser.openFile("./app/codeCharta/assets/logo.png")
+
+		expect(await filePanel.getSelectedName()).toEqual("sample1.cc.json")
```

Comment:
Please run this test in headfull mode and check if an error dialog appears. It might be, that puppeteer ignores the accept attribute and still tries to open a non-json file anyway. If puppeteer doesn't ignore it and it just refuses to upload a file, this would be awesome.

Reply:
Not possible. Approved